app/controllers/api/scores_controller.rb

```diff
@@ -15,6 +15,23 @@ def user_feed
       render json: response.to_json
     end
 
+    def show
+      user = User.find(params[:id])
+      if user
```

Comment:
`User.find` returns the user on success or **directly throws** an error on failure.
Taking this into consideration, do we still need to check `if user` ?

In practice, there are 2 ways to handle this:
1. Use `User.find(...)` and let the error get to the user in a Rails format. In this case, there is no need for checking `if user`, because at that point, `user` can only be a User instance.
2. Use `User.find_by(id: ...)` which returns the user on success or `nil` on failure. In this case, we should further check `if user` or `if user.present?` and handle the case like we are doing right now.

For now, I suggest use the second option, but keep in mind both of the variants.